config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -62,8 +62,6 @@ presubmits:
     max_concurrency: 12
     labels:
       preset-service-account: "true"
-      preset-bazel-scratch-dir: "true"
```

Comment:
don't need dind preset here? (like the other two)

Reply:
it was already here on line 67/65 below